measuring/questions/who-uses.md

```diff
@@ -0,0 +1,15 @@
+# Who uses the InnerSource project?
+
+Depending on the InnerSource project, usage could look something like:
```

Comment:
What does usage refer to?

I presume it means usage of the GQM, and this describes how you would measure it?

Reply:
I'm glad you asked.  This references to usage of _any_ InnerSource project.  If someone has a goal to reduce duplication of work then one question that supports that goal is how widely used are their InnerSource projects.  Let me know if that makes sense?